README.md

```diff
@@ -2,111 +2,6 @@
 
 # Diff output extensions
 
-Pure PHP strict Unified diff format output builder (similar to `diff -u`).
+This branch provides a version for `PHP CS Fixer diff` only.
```

Comment:
wow :D

Reply:
that's what I mean't "it gonna be hard" :D

Reply:
so, it's the same package as main one. do you have any plans to release it? like a tag?

Reply:
yes, I can create tags on this branch for `php-cs-fixer diff`, I think this should work. Although, does composer allow for a configuring a tag different than a version tag (i.e. like `"dev-php-cs-fixer-diff" L "^1.0"`)?

Reply:
in your example `dev-php-cs-fixer-diff` is package, not version. tag is version.